tests/contrib/monitors/mixins/test_validation.py

```diff
@@ -199,16 +221,15 @@ def test_check_fields_errors_percent_multiple_fields():
         monitor.check_fields_errors_percent(field_names=['field2', 'field3'])
 
 
-def test_check_fields_errors_percent_no_fields_old():
-    monitor = DummyValidationMonitor(stats)
-    monitor.new_behavior = False
+def test_check_fields_errors_percent_no_fields_old(old_monitor):
+    monitor.correct_field_list_handling = False
```

Comment:
I think it should be `old_monitor`, right?

Reply:
It's just an unused line :) I'll remove it.